airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -150,7 +150,7 @@
   "credits.emailVerification.resendConfirmation": "We send you a new verification link.",
   "credits.emailVerification.resend": "Send verification link again",
   "credits.lowBalance": "Your credit balance is low. You need to add more credits to prevent your syncs from being stopped.",
-  "credits.zeroBalance": "All your syncs have been stopped because your credit balance is 0. Add more credits to enable your syncs again.",
+  "credits.zeroBalance": "All your paid syncs have been stopped because your credit balance is 0. Add more credits to enable your paid syncs again.",
```

Comment:
@andyjih any thoughts on this wording? I want to make it clear that free connector program syncs will continue.

![image](https://user-images.githubusercontent.com/7550957/216935751-904f4114-a281-4c8f-814a-ac0fdb3555c3.png)

Reply:
Discussed here: https://airbytehq-team.slack.com/archives/C049XDAS19C/p1675775965163619